packages/zapp/console/src/components/LaunchPlan/LaunchPlanVersionDetails.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import { withRouteParams } from 'components/common/withRouteParams';
+import { ResourceIdentifier, ResourceType } from 'models/Common/types';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { WaitForData } from 'components/common/WaitForData';
+import { useProject } from 'components/hooks/useProjects';
+import { LaunchPlanId } from 'models/Launch/types';
+import { entitySections } from 'components/Entities/constants';
+import { EntityDetailsHeader } from 'components/Entities/EntityDetailsHeader';
+import { EntityVersions } from 'components/Entities/EntityVersions';
+
+const useStyles = makeStyles((_theme: Theme) => ({
+  verionDetailsContatiner: {
+    display: 'flex',
+    flexDirection: 'column',
+    flexWrap: 'nowrap',
+    overflow: 'hidden',
+    height: `calc(100vh - ${_theme.spacing(1)}rem)`,
+  },
+  staticGraphContainer: {
+    display: 'flex',
+    height: '60%',
+    width: '100%',
+    flex: '1',
+  },
+  versionsContainer: {
+    display: 'flex',
+    flex: '0 1 auto',
+    height: '40%',
+    flexDirection: 'column',
+    overflowY: 'scroll',
```

Comment:
It looks like we will never have a graph container for launch plans. 
Currently 60%for graph and 40% for version lead to the bug when, all version show only few rows, with a lot white space below and a scroller to see all version in the view provided
<img width="1253" alt="Screen Shot 2022-06-16 at 11 25 56 PM" src="https://user-images.githubusercontent.com/55718143/174238370-a4490105-b71c-4087-84c5-35f54295a490.png">
Looked at `aa.wine_study.wf`

Reply:
fixed